website/versioned_docs/version-0.68/headless-js-android.md

```diff
@@ -57,6 +64,39 @@ public class MyTaskService extends HeadlessJsTaskService {
 }
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+package com.your_application_name;
+
+import android.content.Intent
+import android.os.Bundle
+import com.facebook.react.HeadlessJsTaskService
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.jstasks.HeadlessJsTaskConfig
+
+class MyTaskService : HeadlessJsTaskService() {
+    override fun getTaskConfig(intent: Intent): HeadlessJsTaskConfig? {
+        val extras: Bundle? = intent.extras
+        if (extras != null) {
+            return HeadlessJsTaskConfig(
+                "SomeTaskName",
+                Arguments.fromBundle(extras),
+                5000, // timeout for the task
+                false // optional: defines whether or not  the task is allowed in foreground.
+                // Default is false
```

Comment:
Can we put this in the line above?

Reply:
We can, but `ktfmt` move to the next line :(

Reply:
Let's stick with ktfmt's stlye then